source/frontend/src/features/mapSideBar/compensation/detail/CompensationRequisitionDetailView.tsx

```diff
@@ -130,14 +146,35 @@ export const CompensationRequisitionDetailView: React.FunctionComponent<
     ?.map(f => f.totalAmount ?? 0)
     .reduce((prev, next) => prev + next, 0);
 
-  const acqFileProject = acquisitionFile?.project;
-  const acqFileProduct = acquisitionFile?.product;
-
-  const statusSolver = new StatusUpdateSolver(acquisitionFile);
+  const acqFileProject = file?.project;
```

Comment:
leases can have projects too can't they? just not products (at least for the moment).

Reply:
renamed variable.